--- a/setup.py
+++ b/setup.py
@@ -8,9 +8,6 @@
 with open(os.path.join(APP_PATH, 'resources', 'README.rst')) as f:
       _LONG_DESCRIPTION = f.read()
 
-with open(os.path.join(APP_PATH, 'resources', 'requirements.txt')) as f:
-      _INSTALL_REQUIRES = list(map(lambda s: s.strip(), f.readlines()))
-
 _DESCRIPTION = \
     "An adapter to Linux kernel support for inotify directory-watching."
 
@@ -29,11 +26,9 @@
     packages=setuptools.find_packages(exclude=['tests']),
     include_package_data=True,
     zip_safe=False,
-    install_requires=_INSTALL_REQUIRES,
     package_data={
         'inotify': [
             'resources/README.rst',
-            'resources/requirements.txt',
         ]
     }
 )
